storage/storage_model.go

```diff
@@ -14,6 +14,23 @@ import (
 
 const metadataFilename = "metadata.json"
 
+func parseLogLineString(data string) (model.LogLineItem, error) {
+	ts, err := strconv.ParseInt(strings.TrimSpace(data[3:23]), 10, 64)
+	if err != nil {
+		return model.LogLineItem{}, err
+	}
+
+	return model.LogLineItem{
+		Timestamp: time.Unix(0, ts*1e6).UTC(),
+		// We need to Trim the newline here because logkeeper doesn't expect newlines to be included in the LogLineItem
+		Data: strings.TrimRight(data[23:], "\n"),
+	}, nil
+}
+
+func makeLogLineString(logLine model.LogLine) string {
+	return fmt.Sprintf("  0%20d%s\n", logLine.Time.UnixNano()/1e6, logLine.Msg)
```

Comment:
We have a function [UnixMilli](https://github.com/evergreen-ci/utility/blob/main/time.go#L26) in our `utility` package that will do the conversion for you.

Reply:
It's built in after 1.8.....

Reply:
Fixed. Thanks for the heads up!